sdks/python/apache_beam/transforms/managed.py

```diff
@@ -183,10 +178,8 @@ def _resolve_expansion_service(
     return expansion_service
 
   default_target = None
-  for gradle_target, transforms in _EXPANSION_SERVICE_JAR_TARGETS.items():
-    if transform_name.lower() in transforms:
-      default_target = gradle_target
-      break
+  if identifier in MANAGED_TRANSFORM_URN_TO_JAR_TARGET_MAPPING:
+    default_target = MANAGED_TRANSFORM_URN_TO_JAR_TARGET_MAPPING.get(identifier)
```

Comment:
Might as well simplify and throw an error if identifier is not in `MANAGED_TRANSFORM_URN_TO_JAR_TARGET_MAPPING`

Reply:
Done.